Woof/WoofTests/ViewModel/OwnerProfileViewModelTests/OwnerMainTabViewModelTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+
+final class OwnerMainTabViewModelTests: XCTestCase {
+    func testOwnerMainTabViewModelExistsInAPI() {
+        OwnerMainTabViewModel()
```

Comment:
try to build an app and look for warning messages, you should solve them
```suggestion
        _ = OwnerMainTabViewModel()
```

Reply:
Thanks, sorry for this.
Fixed.